homeassistant/components/ads/__init__.py

```diff
@@ -46,13 +69,88 @@
 
 SERVICE_WRITE_DATA_BY_NAME = "write_data_by_name"
 
+BINARY_SENSOR_DEFAULT_NAME = "ADS binary sensor"
+COVER_DEFAULT_NAME = "ADS Cover"
+LIGHT_DEFAULT_NAME = "ADS Light"
+SENSOR_DEFAULT_NAME = "ADS sensor"
+SWITCH_DEFAULT_NAME = "ADS Switch"
+
+PLATFORMS = (
+    (BINARY_SENSOR_DOMAIN, CONF_BINARY_SENSORS),
+    (COVER_DOMAIN, CONF_COVERS),
+    (LIGHT_DOMAIN, CONF_LIGHTS),
+    (SENSOR_DOMAIN, CONF_SENSORS),
+    (SWITCH_DOMAIN, CONF_SWITCHES),
+)
```

Comment:
Please import and use the new `Platform` enum:
```suggestion
PLATFORMS = (
    (Platform.BINARY_SENSOR, CONF_BINARY_SENSORS),
    (Platform.COVER, CONF_COVERS),
    (Platform.LIGHT, CONF_LIGHTS),
    (Platform.SENSOR, CONF_SENSORS),
    (Platform.SWITCH, CONF_SWITCHES),
)
```